pkg/controller/core.oam.dev/v1alpha2/application/application_controller.go

```diff
@@ -349,6 +371,16 @@ func appWillRollout(app *v1beta1.Application) bool {
 	return len(app.GetAnnotations()[oam.AnnotationAppRollout]) != 0 || app.Spec.RolloutPlan != nil
 }
 
+func hasHealthCheckPolicy(policies []*appfile.Workload) bool {
```

Comment:
Why is the parameter named to `policies`?

Reply:
Because appfile pkg treats `Policy` same as `Workload`, it uses struct `Workload` to represent a Policy. I name the parameter as policies to highlight these entities are truly policies.